Fix attachment download crash

petrspevak opened the pull request:
## Problem
The current AppImage configuration for Synology Chat crashes whenever users try to download attachments (in received or sent message).

## Solution
- Updated .deb packages that were causing crashes during attachment downloads
- Upgraded obsolete .deb packages that were preventing successful AppImage builds

## Changes
- Modified `synochat.json` to use updated package versions
- Ensured all dependencies are compatible with current build requirements
- Added .gitignore to exclude build artifacts and IDE configuration files

## Testing
- [x] AppImage builds successfully
- [x] Attachment downloads work without crashes
- [x] All core functionality remains intact

petrspevak committed:
chore: .gitignore file added

- deb2appimage (external tool) and IDE configurations from version control

petrspevak committed:
fix: prevent crash when downloading attachments

- upgrade .deb packages that caused crashes on every attachment download
- update obsolete .deb packages required for a successful AppImage build

p-fruck commented on synochat.json:
```diff
@@ -6,10 +6,10 @@
             "repoarch": "amd64",
             "distrorepo": "Debian",
-            "repoversion": "bullseye,buster",
+            "repoversion": "bookworm",
```
Thanks for the PR! Does it make sense update to debian trixie right away?

petrspevak replied:
Yes. I will give it a try (+ 1 day test by using created AppImage in our company's communication) and let you know or directly push the change.

petrspevak committed:
chore: distro changed to Ubuntu LTS

 - Ubuntu is the only distro, where you can run SynoChat .deb directly
- deps reduced - resolved by running Ubuntu in Docker container and `dpkg-deb -f synochat.deb`
